### DISCUSSION TITLE
Törvényi megjelenés

### DISCUSSION THREAD

**Discussion by enyugta** [nav-gov-hu/eRECEIPT#45] [Kérdések és válaszok]
Az "Egyes adótörvények módosításáról" szóló törvényjavaslat (T/9724. szám) szerint:

"...az e-Nyugta-rendszerrel összefüggő rendelkezések hatályba lépését 2025. január 1. napjáról 2025. július 1-jére, illetve a szankcionáló rendelkezését (mulasztási bírság) 2025. szeptember 1. napjára elhalassza."

Mikorra várható egy átfogó rendelet az eNyugta rendszerrel kapcsolatban?

**Comment by BatariZ** [comment]
A vonatkozó, időközben megjelent rendelet: A nemzetgazdasági miniszter 8/2025. (III. 31.) NGM rendelete az e-pénztárgépek forgalmazásáról, üzemeltetéséről, valamint az e-pénztárgépek és az e-nyugta kiállításának követelményeiről
